_docs/tutorial/00-setup.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Setup
+overview: The required setup instructions for the tutorial.
+
+order: 00
+
+layout: docs
+type: markdown
+---
+{% include home.html %}
+
+1. Install [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git), [curl](https://curl.haxx.se/download.html), [node.js](https://nodejs.org/en/download/), [Docker](https://docs.docker.com/install/)
+and get access to a [Kubernetes](https://kubernetes.io) cluster.
+For example, you can try the [IBM Cloud Container Service](https://console.bluemix.net/docs/containers/container_index.html#container_index).
+
+1. Follow the [Prerequisites of the Kubernetes Quick Start instructions]({{home}}/docs/setup/kubernetes/quick-start.html#prerequisites).
+
+1. Follow the steps 1-4 of the [Installation Steps of the Kubernetes Quick Start instructions]({{home}}/docs/setup/kubernetes/quick-start.html#installation-steps).
+
+1. This tutorial assumes that you perform the commands of the steps in the Istio directory that you downloaded and extracted in the steps 1 and 2 of the [Installation Steps of the Kubernetes Quick Start instructions]({{home}}/docs/setup/kubernetes/quick-start.html#installation-steps).
+
+1. Download Istio sources into the Istio directory. **Note** that Istio source code is under [Apache 2.0](https://www.apache.org/licenses/LICENSE-2.0) license.
```

Comment:
Why are the sources needed? It should work with just the release tgz
the review of the sources could just be links into github

I don't think "git clone..." should be part of user facing instructions - it can only lead to issue with mismatch between release version and state of master etc


the install istio step for instance will not work from source

Reply:
Good point. The sources are needed for the only one reason: to run _ratings_ as a node.js application, and as a Docker container. So basically, only https://github.com/istio/istio/tree/master/samples/bookinfo/src/ratings is needed. I will instruct the users to download the source code from https://github.com/istio/istio/releases, so the source code will always match the release.

Reply:
or you could just point them https://github.com/istio/istio/blob/master/samples/bookinfo/src/ratings/ratings.js as it's just one file ?
or wget -r https://github.com/istio/istio/blob/master/samples/bookinfo/src/ratings/
if you also want the Dockerfile etc (why would you ?)

Reply:
I changed the setup to instruct to download the sources from https://github.com/istio/istio/releases, please check.
The sources are already packaged there, they match the release, no need to install `wget`.

I need the Docker file for the step 2, Running a single microservice in a Docker container.